source/overview/security.rst

```diff
@@ -28,49 +28,59 @@ Private Cloud Deployment with Secure Mobile Apps
+   - Session length, session cache, and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`__, automatically forcing a user to re-login after a specified period of time.
+   - Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`__ across web, mobile devices, and native desktop apps.
    - Remotely reset user passwords via the System Console or via the `command line <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-password>`__.
    - Mattermost supports integrated authentication with `Active Directory and LDAP <https://docs.mattermost.com/deployment/sso-ldap.html>`__ (E10) as well as `SAML 2.0 SSO integration <https://docs.mattermost.com/deployment/sso-saml.html>`__ with providers including `Active Directory Federation Services <https://docs.mattermost.com/deployment/sso-saml-adfs.html>`__,  `Okta <https://docs.mattermost.com/deployment/sso-saml-okta.html>`__, among others (E20).
    - The ability to require `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`__ is also available (E10).
 
 Access Control Policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+If a user logins in and enters the correct email address but wrong password a "your password is incorrect" response is presented. This can allow account enumeration. This is a deliberate design decision we've made for the product in the context of overall system security. 
+
+Security is a top concern for organizations who deploy Mattermost in a private network. While running under existing policies and auth protocols of the private network, we prioritize a better user experience (telling a user what went wrong on login) ahead of preventing users on the private network from enumerating user accounts. 
+
+Mattermost is optimized to be deployed in a highly secure environment. However, admins are given a choice as to the deployment environment. 
+
+When Mattermost is deployed outside a firewall, Admins must be aware that their system is exposed to issues inherent in public cloud deployments. In this case, we generally follow the standards of leading online services. For example, Gmail offers APIs that not only confirm whether an email account exists, but also displays user's profile picture by default. 
+
+Mattermost has gone through extensive penetration testing and security reviews and there's a long list of [security updates](http://about.mattermost.com/security-updates/) we've made thanks to our community and customers. You can find further details and previous discussion [in our GitHub thread](https://github.com/mattermost/platform/issues/4321#issuecomment-258832013). In addition, the following policies are provided: 
```

Comment:
"long list" doesn't sound good to me here. 

Also, not all of the security updates come from community/customer.

Link formatting is not rendering properly.

Reply:
```suggestion
Mattermost undergoes extensive penetration testing, security reviews, and _`security updates <http://about.mattermost.com/security-updates>`__. You can find further details and previous discussion _`in our GitHub thread <https://github.com/mattermost/platform/issues/4321#issuecomment-258832013>`__. In addition, the following policies are provided: 
```